src/js/disability-benefits/components/AddFilesForm.jsx

```diff
@@ -105,12 +105,18 @@ class AddFilesForm extends React.Component {
           <div key={index} className="document-item-container">
             <Element name={`documentScroll${index}`}/>
             <div className="document-title-size">
-              <div className="document-title-header">
-                <h4 className="title">{file.name}</h4>
+              <div className="document-title-text-container">
+                <div className="document-title-header">
+                  <h4 className="title">{file.name}</h4>
```

Comment:
What happens if the file name is long enough to overlap the button? Does it wrap and look ok?